installer/conf/omsagent.d/mysqlworkload.conf

```diff
@@ -0,0 +1,13 @@
+# This file is an example of what the config file pushed by the server should look like
```

Comment:
separate the words by underscore or dashes? Any recommendation regarding the conf file name?